### PR TITLE
containers ebook takeover

### PR DESCRIPTION
**Pull request by deadlight**
## Done

New homepage takeover 

## QA

Check the takeover on the homepage against the [brief](https://docs.google.com/document/d/1_-Ns4VFpnW8BPbf7cLfQdobWxDhi-_LO6czJ-wA4f7U/edit#) and [design](https://www.dropbox.com/sh/3hld9oy7350bgnw/AACUYs551xaXSSma1bG6n9qJa/Exportables?dl=0&preview=Containers-takeover-desktop.jpg).

### CODE REVIEW

File: static/css/section/_homepage.scss [hunk 1]
```diff
@@ -2139,7 +2139,7 @@ body.homepage-advantage {
     padding-bottom: 20px;
 
     @media only screen and (min-width : $breakpoint-medium) {
-      background-image: url('https://assets.ubuntu.com/v1/f4cb93a3-Yakkety_Yak-Wallpaper_aubergine.jpg');
+      background-image: url('#{$asset-server-url}f4cb93a3-Yakkety_Yak-Wallpaper_aubergine.jpg');
```
Comment (nottrobin) [inline static/css/section/_homepage.scss]: :heart:

**Comment by anthonydillon** [comment]
@deadlight could you please provide a demo link or screenshot for design review please. @yaili @joanasa89

**Comment by deadlight** [comment]
@anthonydillon we're having problems with the demo builder. I'll contact @joanasa89 and @yaili separately.

**Comment by deadlight** [comment]
@joanasa89 @yaili The demo is here: http://www.ubuntu.com-containers-ebook-takeover.demo.haus/

Please take a look ASAP, it's waiting to go live.

**Comment by joanasa89** [comment]
Looks good to me.

**Pull request merged**